### PR TITLE
Fix typos

### PR DESCRIPTION
**Pull request by kaakaa**
#### Summary
Fix typos.

#### Ticket Link
N/A

#### Checklist
- [x] Includes text changes and localization file ([.../i18n/en.json](https://github.com/mattermost/mattermost-webapp/blob/master/i18n/en.json)) updates

### CODE REVIEW

File: i18n/en.json [hunk 1]
```diff
@@ -963,7 +963,7 @@
   "admin.service.sessionDaysEx": "E.g.: \"30\"",
   "admin.service.sessionIdleTimeout": "Session Idle Timeout (minutes):",
   "admin.service.sessionIdleTimeoutDesc": "The number of minutes from the last time a user was active on the system to the expiry of the user's session. Once expired, the user will need to log in to continue. Minimum is 5 minutes, and 0 is unlimited.<br/><br/>Applies to the desktop app and browsers. For mobile apps, use an EMM provider to lock the app when not in use. In High Availability mode, enable IP hash load balancing for reliable timeout measurement.",
-  "admin.service.sessionIdleTimeoutEx": "Ex \"60\"",
+  "admin.service.sessionIdleTimeoutEx": "E.g.: \"60\"",
```
Comment (saturninoabril) [inline i18n/en.json]: I think here we're still not consistent in our current codebase.

`Ex` and `E.g.:` are being used elsewhere even with the same translation id:
- `Ex` at component https://github.com/mattermost/mattermost-webapp/blob/master/components/admin_console/session_settings.jsx#L94
- `E.g.:` at https://github.com/mattermost/mattermost-webapp/blob/master/i18n/en.json#L963

@jasonblais Should we change those `Ex` into `E.g.:` in components for all occurrences of `en` translations?

@kaakaa Would you like to include those as well?

Comment (jasonblais) [reply i18n/en.json]: @saturninoabril Yes, we should use `E.g.` everywhere for consistency.

Comment (kaakaa) [reply i18n/en.json]: @saturninoabril Thanks, I forgot to fix codebase.
Fixes were done.

And by grepping simply, there are 62 messages with `Ex` still be remain in codebase.
```
$ find ./ -name "*.jsx" | xargs grep "'Ex " | wc -l
62
```

Should fixes for the rest of that messages be done by this PR? or another PR?

Comment (saturninoabril) [reply i18n/en.json]: Thanks @kaakaa! I suggests to have it here since most of typo fixes here are similar.

Comment (kaakaa) [reply i18n/en.json]: The rest was fixed.

Comment (saturninoabril) [reply i18n/en.json]: Thanks @kaakaa!

**Pull request merged**